bitnami/joomla/Chart.yaml

```diff
@@ -1,22 +1,30 @@
-apiVersion: v1
-name: joomla
-version: 8.1.10
+annotations:
+  category: CMS
+apiVersion: v2
 appVersion: 3.9.22
+dependencies:
+- condition: mariadb.enabled
+  name: mariadb
+  repository: https://charts.bitnami.com/bitnami
+  version: 9.x.x
+- name: common
+  repository: https://charts.bitnami.com/bitnami
+  version: 1.x.x
 description: PHP content management system (CMS) for publishing web content
-keywords:
-  - joomla
-  - cms
-  - blog
-  - http
-  - php
+engine: gotpl
 home: https://github.com/bitnami/charts/tree/master/bitnami/joomla
 icon: https://bitnami.com/assets/stacks/joomla/img/joomla-stack-220x234.png
-sources:
-  - https://github.com/bitnami/bitnami-docker-joomla
-  - http://www.joomla.org/
+keywords:
+- joomla
+- cms
+- blog
+- http
+- php
 maintainers:
-  - name: Bitnami
-    email: containers@bitnami.com
-engine: gotpl
-annotations:
-  category: CMS
+- email: containers@bitnami.com
```

Comment:
I would say this indentation level will not pass the linter. I should be two spaces. The same happens in all the arrays of this file

Reply:
Yep, I think so. Running the following commands everything should be prettified as expected
```
$ yq r --tojson Chart.yaml | jq . | yq r --prettyPrint - > Chart.yaml.new
$ mv Chart.yaml.new Chart.yaml
```